android-components/components/feature/fxsuggest/src/main/java/mozilla/components/feature/fxsuggest/facts/FxSuggestFacts.kt

```diff
@@ -26,6 +26,7 @@ class FxSuggestFacts {
      */
     object MetadataKeys {
         const val CLICK_INFO = "click_info"
+        const val IMPRESSION_INFO = "impression_info"
```

Comment:
These might want to move to `FxSuggestSuggestionProvider.SuggestionMetadataKeys`, or somewhere like that, since they're passed in `AwesomeBar.Suggestion.metadata`, not `Fact.metadata`—but I wanted to minimize churn for the first review pass.